build_projects/dotnet-cli-build/DotNetMSBuildTool.cs

```diff
@@ -40,5 +42,10 @@ private string GetVerbosityArg()
 
             return null;
         }
+
+        private string GetAdditionalParameters()
+        {
+            return AdditionalParameters;
+        }
```

Comment:
What's the point of this method? Why not just use the property?

Reply:
I was trying to follow the pattern I saw in another similar class.  I've updated it now.